bears/markdown/MarkdownBear.py

```diff
@@ -101,7 +101,10 @@ def create_arguments(filename, file, config_file,
             "ruleRepetition": markdown_horizontal_rule_repeat,  # int
         }
 
-        config_json = json.dumps(remark_configs)
-        # Remove { and } as remark adds them on its own
-        settings = config_json[1:-1]
-        return '--no-color', '--quiet', '--setting', settings
```

Comment:
wait - did this actually work? At least this commit seems to be not helpful, config_file should be None all the time